src/main/java/com/mixpanel/android/viewcrawler/ViewVisitor.java

```diff
@@ -282,6 +251,51 @@ private boolean verifyLayout(View target, Set<Integer> rules) {
                 }
             }
 
+            int size = mArgs.size();
+            for (int i = 0; i < size; i++) {
+                LayoutRule layoutRule = mArgs.get(i);
+                final int viewId = layoutRule.viewId;
```

Comment:
We may not need the `viewId`, `newVerb`, and `newAnchorId` variables, since you can just refer directly to the members of `layoutRule`